examples/pytest/dummy/conftest.py

```diff
@@ -0,0 +1,66 @@
+##########################################################################
+# Copyright (c) 2010-2022 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
```

Comment:
Did you handled the case where we have env variables?
Basically https://kiso-testing.readthedocs.io/en/latest/getting_started/user_guide.html#test-configuration-file should all be considered